openshift/e2e-tests-openshift.sh

```diff
@@ -154,7 +164,11 @@ function run_e2e_tests(){
   # Add anyuid scc to all authenticated users so e2e tests for --user flag can user any user id
   oc adm policy add-scc-to-group anyuid system:authenticated
 
-  go_test_e2e -timeout=$E2E_TIMEOUT -mod=vendor ./test/e2e || fail_test
+  report_go_test \
+    ./test/e2e \
+    -v -timeout=$E2E_TIMEOUT -mod=vendor \
+    -tags="e2e $TAGS" || fail_test
```

Comment:
if `tags` flag is passed with multiple values before e2e test dir, it wont work as it tries to find the tests in second tag value passed

```
+ go test -v -race -count=1 -tags=e2e serving -timeout=60m -mod=vendor ./test/e2e
+ tee /tmp/tmp.I1d4iZVLju
can't load package: package serving: cannot find package "." in:
	/go/src/github.com/knative/client/vendor/serving
```
ref: [log](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_knative-client/351/pull-ci-openshift-knative-client-release-next-e2e-aws-ocp-43/219#1:build-log.txt%3A561)

Reply:
ideally, this should be fixed in test-infra lib function `go_test_e2e` where the args should be processed and test dir should be sorted as first arg followed by rest of the flags

Reply:
quoting wont work as its passed as args to `report_go_test`

Reply:
filed the issue in test-infra 2064